nox/sessions.py

```diff
@@ -360,7 +360,11 @@ def _run(
         return nox.command.run(args, env=env, paths=self.bin_paths, **kwargs)
 
     def conda_install(
-        self, *args: str, auto_offline: bool = True, **kwargs: Any
+        self,
+        *args: str,
+        auto_offline: bool = True,
+        channel: Union[str, Sequence] = "",
```

Comment:
```suggestion
        channel: Union[str, Sequence[str]] = "",
```

Bare `Sequence` is equivalent to `Sequence[Any]`. Should we be more specific here?

Reply:
Yes, Sequence[str] is better.